README.md

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func main() {
-	c := hipchat.NewClient("<PUT YOUR AUTH TOKEN HERE>")
```

Comment:
This is correct syntax (https://github.com/andybons/hipchat/blob/master/hipchat.go#L86)

Are you getting an error?